solr/core/src/java/org/apache/solr/util/circuitbreaker/CircuitBreakerManager.java

```diff
@@ -121,19 +135,41 @@ public static String toErrorMessage(List<CircuitBreaker> circuitBreakerList) {
    *
    * Any default circuit breakers should be registered here.
    */
-  public static CircuitBreakerManager build(SolrConfig solrConfig) {
-    CircuitBreakerManager circuitBreakerManager = new CircuitBreakerManager(solrConfig.useCircuitBreakers);
-
-    // Install the default circuit breakers
-    CircuitBreaker memoryCircuitBreaker = new MemoryCircuitBreaker(solrConfig);
-    CircuitBreaker cpuCircuitBreaker = new CPUCircuitBreaker(solrConfig);
+  @SuppressWarnings({"rawtypes"})
```

Comment:
can we avoid adding new suppress to the code please?

Reply:
Unfortunately, that is the way NamedArgs are structured in PluginInfo -- we need to fix that before we remove this warning. I will follow up.